Arlo.py

```diff
@@ -620,11 +633,13 @@ def DownloadRecording(self, url, to):
     # You will need the to install a library to handle streaming of this protocol: https://pypi.python.org/pypi/python-librtmp
     #
     # The request to /users/devices/startStream returns:
-    #{ "url":"rtmps://vzwow09-z2-prod.vz.netgear.com:80/vzmodulelive?egressToken=b1b4b675_ac03_4182_9844_043e02a44f71&userAgent=web&cameraId=48B4597VD8FF5_1473010750131" }
+    #{ "url":"rtsp://<url>:443/vzmodulelive?egressToken=b<xx>&userAgent=iOS&cameraId=<camid>" }
     #
     ##
     def StartStream(self, camera):
```

Comment:
I wonder if we should change this method name to `GetStreamUrl` and make it just return the url instead of the dictionary...

I had originally intended to write a function that actually grabbed the stream and returned it in the form of a generator, but never had the time to figure it out.

Reply:
Agree on both fronts. Will push code for cancelling the Timer thread and and changing the stream method.